### ISSUE TITLE
Fig seems to mess with background process handling in Nodemon.

### ISSUE THREAD

**Issue by davesters** [withfig/fig#462]
### Description:

Fig seems to mess with whatever process manager that is used by `nodemon`. I've been running my node apps locally with nodemon during development in the VSCode Terminal. When nodemon goes to restart the process when I modify files, more than half the time it doesn't kill the process properly, so when it tries to restart it, the port is used so it fails. I have to manually kill the process and restart nodemon.

This never happened before, but started happening after I installed Fig. I turned Fig off and it appears to have stopped happening. I don't have much more information than that at this time.

Let me know if you need me to run any debug tools or get more information.
 
### Details:
|macOS|Fig|Shell|
|-|-|-|
|11.5.2|Version 1.0.48 (B264)|/bin/zsh|
<details><summary><code>fig diagnostic</code></summary>
<p>
<pre>Version 1.0.48 (B264) [U.S.] 
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: ~/.fig/bin/fig
Accessibility: true
Number of specs: 196
SSH Integration: false
Tmux Integration: true
Keybindings path: ~.fig/user/keybindings
iTerm Integration: true [Authenticated]
Hyper Integration: false
VSCode Integration: true
Docker Integration: true
Symlinked dotfiles: false
Only insert on tab: false
Installation Script: true
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: ??? (???) - ???
Current working directory: ???
Current window identifier: ???</pre>
</p>
</details>

**Comment by mschrage** [comment]
Hey @davesters! So sorry you're running into this issue. To be honest, I'm not totally sure the mechanism that could be causing this.

1. How do you launch nodemon?

2. How did you turn Fig off? Just by quitting the app or by deleting it?

**Comment by davesters** [comment]
I quit it by just qutting the app (not deleting it).

However, I was about to report back in a bit. I saw it happen again even without Fig running, so now I am not so sure it is because of Fig. I made sure Fig was not running and restarted VS Code and haven't seen it again yet. I'll keep going about my normal business and run it with and without fig for longer and see what happens. Probably safe to close this issue for now.

Thanks for the quick response.
